### DISCUSSION TITLE
96

### DISCUSSION THREAD

**Discussion by giscus** [CODe5753/giscus#7] [Comments]
# 96

성과 프로젝트 성과 1학기 관통 프로젝트 우수상(2등) 2학기 공통 프로젝트 우수상(2등) 2학기 공통 프로젝트 UCC 우수상(4등) 2학기 특화 프로젝트 SSDC 오픈소스 삼성 리서치 우수상(1등) 2학기 특화 프로젝트 UCC 우수상(2등) 그 외 성과 오픈소스 기여 SSDC에 프로젝트 소개 좋은 사람들을 너무 많이 얻음😀 여러 기술 습득 중도 퇴소 수료까지 단 1개월을 앞둔 상황에서, 많은 고민 끝에 중도 퇴소를 결심했습니다. 갑작스러운 퇴소+방향성에 대해 큰 고민을 안고 있었는데, 2학기 컨설턴트님 두 분과 실습 코치님 한 분께서 너무 친절하고 자세하게 상담을 해주셔서 정말 감사했습니다. 🙏🏻 SSAFY 교육과정 짧은 후기 SSAFY에서 교육을 받으며 총 3개의 프로젝트를 진행했습니다. 1학기 교육..

https://imksh.com/96

**Comment by dev-yesam** [comment]
안녕하세요. 혹시 취업하고도 입사 예정이 1월이라면, ssafy 수료까지 다녀도 괜찮은가요?

**Reply by CODe5753** [reply]
@dev-yesam 안녕하세요
저는 너무 오래전에 수료를해서 정확하지 않을 수 있습니다만, 말씀하신 경우에 문제가 없어 보입니다 :)
정확한 정보는 사무국을 통해 문의해보셔요!
감사합니다

**Reply by dev-yesam** [reply]
아 사무국에 문의해 볼게요! 감사합니다!
